server/test/api/transferts.js

```diff
@@ -0,0 +1,150 @@
+/* global describe, it, beforeEach */
+var chai = require('chai');
+var expect = chai.expect;
+
+/** import test helpers */
+var helpers = require('./helpers');
+var uuid    = require('../../lib/guid');
+helpers.configure(chai);
+
+
+/**
+* The /transferts API endpoint
+*
+* This test suite implements all GET and POST requests on the /projects HTTP API endpoint.
+* NOTE: to run correctly this test please run the `server/models/test/update.sql`
+* and we must have the exchange rate of current date defined
+*/
+describe('The /transferts API endpoint :: ', function () {
+  'use strict';
+
+  var agent = chai.request.agent(helpers.baseUrl);
+
+  /** login before each request */
+  beforeEach(helpers.login(agent));
+
+  /** Test with the current date */
+  var date = new Date();
+
+  /** Primary Cash Object */
+  var primaryCash = {
+      uuid        : uuid(),
+      project_id  : 1,
+      type        : 'E',
+      date        : date,
+      currency_id : 2,
+      account_id  : 3627,
+      cost        : 777,
+      description : 'Test Caisse Transfert/' + date.toString(),
+      cash_box_id : 2,
+      origin_id   : 1,
+      user_id     : 1
+  };
+
+  var wrongPrimaryCash = {
+    uuid        : uuid(),
+    project_id  : 1,
+    type        : 'X',
+    date        : date,
+    currency_id : 2,
+    account_id  : 3627,
+    cost        : 555
+  };
+
+  /** Expected keys */
+  var expectedKeys = Object.keys(primaryCash);
+  expectedKeys.push('document_uuid');
+
+  it('POST /transferts should create a new transfert records in primary_cash, primary_cash_item and journal', function () {
+    return agent.post('/transferts')
+      .send(primaryCash)
+      .then(function (res) {
+        expect(res).to.have.status(201);
+      })
+      .catch(helpers.handler);
+  });
+
+  it('GET /transferts should return a list of all transferts', function () {
+    return agent.get('/transferts')
+      .then(function (res) {
+        expect(res).to.have.status(200);
+        expect(res.body).to.not.be.empty;
+        expect(res.body[0]).to.have.all.keys(expectedKeys);
+
+        /**
+        * @fixme: The `date` column in primary cash must be in `datetime` format
```

Comment:
Good FIXME!  If this needs to be an issue, feel free to create it.

By the way, you can now test date equality in Chai tests using `chai-datetime` that is automatically included in the `helpers` library.  Here is how it looks:
```js
expect(res.body).to.be.json;
expect(res.body.id).to.exist;
expect(res.body.date).to.equalDate(submittedValue.date);  // asserts that the dates are equal
```